### PR TITLE
Binutils 2.30

### PR DESCRIPTION
**Pull request by stikonas**

### CODE REVIEW

File: sysa/bc-1.07.1/bc-1.07.1.sh [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# SPDX-FileCopyrightText: Andrius Štikonas <andrius@stikonas.eu>
```
Comment (fosslinux) [inline sysa/bc-1.07.1/bc-1.07.1.sh:1]: copyright year?

File: sysa/ed-1.4/ed-1.4.sh [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# SPDX-FileCopyrightText: Andrius Štikonas <andrius@stikonas.eu>
```
Comment (fosslinux) [inline sysa/ed-1.4/ed-1.4.sh:1]: same for copyright year

File: sysa/run.sh [hunk 3]
```diff
@@ -193,58 +193,6 @@ build autoconf-2.69
...
-    go_sysb
-fi
-
-# In chroot mode transition directly into System C.
-SYSC=/sysc_image
-sys_transfer "${SYSC}" /sysc gzip patch
-if [ "${CHROOT_ONLY_SYSA}" != True ]; then
-    exec chroot "${SYSC}" /init
-fi
+exec env -i PATH="${PATH}" PREFIX="${PREFIX}" LIBDIR="${LIBDIR}" SOURCES="${SOURCES}" DISTFILES="${DISTFILES}" DESTDIR="${DESTDIR}" SRCDIR="${SRCDIR}" bash run2.sh
```
Comment (fosslinux) [inline sysa/run.sh:198]: Hmm, if we are doing this in sysa now, it might make more sense to do the `.env` stuff that we use in sysc in sysa as well.

Comment (fosslinux) [reply sysa/run.sh:198]: This doesn't have to be in this PR though

**Review by fosslinux — APPROVED** [review]
Generally this looks fine to me. Just the copyright years before merge, but let's do the .env stuff soon too.

File: sysa/run.sh (hunk 3 above)
Comment (stikonas) [reply sysa/run.sh:198]: Yes, I need to redo it for sysa but that PR was already quite large.

**Pull request merged**